src/styles/text.css

```diff
@@ -22,6 +22,7 @@
   --fontWeight-regular: 400;
   --fontWeight-book: 450;
   --fontWeight-medium: 500;
+  --fontWeight-semibold: 600;
```

Comment:
We should let design know that we are adding a new font-weight, and then add in all design system files like tailwind config, styled-component themes, add new `--font-` permutations, etc.

Reply:
Can happen in a separate PR